.eslintrc.json

```diff
@@ -1,24 +1,23 @@
 {
-  "extends": [
-    "./node_modules/kyt/config/.eslintrc.base.json"
-  ],
+    "extends": [
+        "./node_modules/kyt/config/.eslintrc.base.json"
+    ],
 
-  "env": {
-  },
+    "env": {
+    },
 
-  "plugins": [
-  ],
+    "plugins": [
+    ],
 
-  "rules": {
-      "react/prefer-es6-class": 0,
-      "react/prefer-stateless-function": 0,
-      "object-shorthand": 1,
-      "comma-dangle": 0,
-      "indent": ["error", 2, { "SwitchCase": 2, "VariableDeclarator": 2, "outerIIFEBody": 2 }],
-      "react/jsx-indent-props": ["error", 2],
-      "react/jsx-indent": ["error", 2],
-      "object-shorthand": ["off", "always"],
-      "func-names": ["off", "always"]
-
-  }
+    "rules": {
+        "react/prefer-es6-class": 1,
+        "react/prefer-stateless-function": 0,
+        "object-shorthand": 1,
+        "comma-dangle": 0,
+        "indent": ["error", 4, { "SwitchCase": 1, "VariableDeclarator": 1, "outerIIFEBody": 1 }],
+        "react/jsx-indent-props": ["error", 4],
+        "react/jsx-indent": ["error", 4],
+        "object-shorthand": ["off", "always"],
```

Comment:
Object shorthand allows you to do the following:

```js
// es5
var foo = {
    doSomething: () {
        console.log('I did something!');
    }
};

// es6
var foo = {
    doSomething () {
        console.log('I did something!');
    }
};
```

http://eslint.org/docs/rules/object-shorthand